core/jazz_ui/e2e/04-jazzservices/jazzservices.e2e-spec.ts

```diff
@@ -48,15 +48,18 @@ describe('Overview', () => {
             browser.wait(EC.elementToBeClickable(jazzServices_po.getCreateService()), timeOutHigh);
             winhandle = browser.getWindowHandle();
             jazzServices_po.getCreateService().click();
-            servicename='servicename' + Date.now();
+            var min = 111111111;
```

Comment:
why not use something like Math.min and math.max

Reply:
We are using min and max for the random generation. This is just a declaration for that.
Ref:  var randomNum = Math.floor(Math.random() * (Math.max() - Math.min() + 1)) + Math.min();